incubator/redis-auto-cluster/values.yaml

```diff
@@ -0,0 +1,53 @@
+nameOverride: ""
+serviceNameOverride: ""
```

Comment:
Remove `serviceNameOverride`. It's not used. You may add `fullnameOverride`, though.